src/main/java/org/dependencytrack/persistence/ProjectQueryManager.java

```diff
@@ -388,8 +413,12 @@ public Project createProject(String name, String description, String version, Li
         project.setName(name);
         project.setDescription(description);
         project.setVersion(version);
-        if (parent != null) {
-            project.setParent(parent);
+        if (parent != null ) {
+            if (!Boolean.TRUE.equals(parent.isActive())){
+                throw new IllegalArgumentException("An inactive Parent cannot be selected as parent");
+            }else {
+                project.setParent(parent);
+            }
```

Comment:
```suggestion
            }
            project.setParent(parent);
```
Kannst dir das else sparen wenn im if eine exception fliegt